### ISSUE TITLE
new wallet created in vulnerable state

### ISSUE THREAD

**Issue by ixje** [CityOfZion/neo-python#619]
### Current behavior
Created a new wallet, closed neo-python, opened neo-python, tried to open the new wallet, I get the message:

> neo> open wallet retrieve_issue_wallet
> Could not open wallet: This wallet is currently vulnerable. Please execute the "reencrypt_wallet.py" script on this wallet before continuing
> 

### Expected behavior

new wallets should be in the final secure state from the moment we create them

### How to reproduce
```
neo> create wallet retrieve_issue_wallet
[password]> **********
[password again]> **********
```

The help message can also be improved, as people might not know that the `reencrypt_wallet.py` script is located under `./neo/bin/`
### Your environment

Let us know in what environment you're running into the issue:

- OS: OSX high sierra
- neo-python version: 0.7.8-dev
- Python version: (hint: `python -V`) 3.6.5

**Comment by localhuman** [comment]
This issue was recently fixed, and I can't currently reproduce on dev.  

Also, the wallet's were not created in a vulnerable state, but the persisted flag indicating it is not vulnerable was not being persisted.

**Comment by ixje** [comment]
I was in a rush for an appointment so didn't really investigate in depth, but I just tried again and it's not working 🤔 

```
(venv) Eriks-Air:neo-python erik$ git show
commit 951de3a902a398ce6b178481e837ce96014affbd (HEAD -> CityOfZion/development, CityOfZion/development)
Merge: cfc8361 0cc9c03
Author: localhuman <tasaunders@gmail.com>
Date:   Tue Sep 18 10:20:29 2018 +0200

    Merge pull request #611 from CityOfZion/persist-reset-fix

    Persist reset fix

(venv) Eriks-Air:neo-python erik$ python neo/bin/prompt.py
[I 180919 15:05:32 LevelDBBlockchain:115] Created Blockchain DB at /Users/erik/.neopython/Chains/SC234
[I 180919 15:05:40 NotificationDB:73] Created Notification DB At /Users/erik/.neopython/Chains/Test_Notif
NEO cli. Type 'help' to get started


neo> create wallet test_wallet
[password]> **********
[password again]> **********
[I 180919 15:05:53 UserWallet:193] contract does not exist yet
[I 180919 15:05:53 UserWallet:539] Script hash b'E\xbb:a<\xd0\xd9\xfeo\xeb\x9c\xc2^\xf4oY\xa4x\x07\xf5' <class 'bytes'>
Wallet {
    "path": "test_wallet",
    "addresses": [
        {
            "address": "AN8aZZrnCoCBze7tnn95ySKvGc7s7eLejD",
            "script_hash": "45bb3a613cd0d9fe6feb9cc25ef46f59a47807f5",
            "tokens": null
        }
    ],
    "height": 0,
    "percent_synced": 0,
    "synced_balances": [],
    "public_keys": [
        {
            "Address": "AN8aZZrnCoCBze7tnn95ySKvGc7s7eLejD",
            "Public Key": "03835e5c5ff9582b893aad07de0d10b559c48371c9d8de95f7ee6001cd70794d6d"
        }
    ],
    "tokens": [],
    "claims": {
        "available": "0.0",
        "unavailable": "0.0"
    }
}
Pubkey b'03835e5c5ff9582b893aad07de0d10b559c48371c9d8de95f7ee6001cd70794d6d'
neo> exit
Shutting down. This may take a bit...
Closed wallet test_wallet
(venv) Eriks-Air:neo-python erik$ python neo/bin/prompt.py
[I 180919 15:06:14 LevelDBBlockchain:115] Created Blockchain DB at /Users/erik/.neopython/Chains/SC234
[I 180919 15:06:22 NotificationDB:73] Created Notification DB At /Users/erik/.neopython/Chains/Test_Notif
NEO cli. Type 'help' to get started


neo> open wallet test_wallet
Could not open wallet: This wallet is currently vulnerable. Please execute the "reencrypt_wallet.py" script on this wallet before continuing
```
and the `reencrypt` script also doesn't seem to work

```
neo> exit
Shutting down. This may take a bit...
(venv) Eriks-Air:neo-python erik$ python neo/bin/reencrypt_wallet.py test_wallet
[new password]> **********
[new password again]> **********
Looking for key MigrationState
The wallet is vulnerable, will proceed with the operation.
Looking for key MasterKey
Looking for key PasswordHash
Looking for key IV
A new wallet was created with your master key encrypted with the new password, with the name new_test_wallet. You can now open this new wallet with the new version of neo-python
(venv) Eriks-Air:neo-python erik$ python neo/bin/prompt.py
[I 180919 15:07:56 LevelDBBlockchain:115] Created Blockchain DB at /Users/erik/.neopython/Chains/SC234
[I 180919 15:08:03 NotificationDB:73] Created Notification DB At /Users/erik/.neopython/Chains/Test_Notif
NEO cli. Type 'help' to get started


neo> wallet open new_test_wallet
Please open a wallet
neo> open wallet new_test_wallet
Could not open wallet: This wallet is currently vulnerable. Please execute the "reencrypt_wallet.py" script on this wallet before continuing
```

**Comment by ixje** [comment]
will close for now as I can't seem to reliable reproduce it. First 4-5 tries it failed, now when I create a new wallet it works fine 🤔
